### PR TITLE
change text to state that devops is awesome

### PR DESCRIPTION
**Pull request by tnamba-f5**

### CODE REVIEW

**Comment by gregba17** [comment]
done

**Comment by patidarashok** [comment]
Done by ashok

**Comment by dhivyab90** [comment]
Devops

**Comment by bahram663** [comment]
new devops

File: src/index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 </head>
 <body style="background-color: #1A729D;">
     <h1 style="font-family: Courier; color: #434342; font-size: 48px; text-align: center;">
-        DevOps is great!
+        DevOps is awesome!
```
Comment (juliusbgit) [inline src/index.html:8]: Ericawesome

**Comment by Rajsin01** [comment]
Devops is awesome

**Comment by oviyan** [comment]
Devops is awesome

**Comment by blesson2001** [comment]
csdfsdsd

File: src/index.html (hunk 1 above)
Comment (jeetkarkee) [inline src/index.html:8]: DepOps rocks!

**Comment by vinniechhabra** [comment]
ff

**Comment by vinniechhabra** [comment]
jj

**Review by vinniechhabra — COMMENTED** [review]
g

**Comment by vegiops** [comment]
ff

**Comment by mmrasmussen** [comment]
ff

**Comment by hakanatak** [comment]
i see that !

**Comment by misgav777** [comment]
werw

**Review by dragonskies — COMMENTED** [review]
check

**Comment by nomadicpeople** [comment]
check

**Comment by rswain06782-art** [comment]
DevOps is Awsome

**Pull request closed without merging**